find_primes.py

```diff
@@ -1 +1,29 @@
+#num1 is the original number, num2 is the prime number, num3 is all of the primes,
+num1 = int(input("Enter any positive integer: "))
+num2 = num1 + 1
+num3 = []
+count = 0
```

Comment:
We should try to use variable names that are descriptive and self-documenting, so that it’s obvious what they are used for when people see them. So instead of num1, num2, num3, and count, how about input_number, starting_number, found_primes, and number_of_primes_found, instead?